### ISSUE TITLE
Vi Mode Indicator show exception

### ISSUE THREAD

**Issue by Fengjing95** [jeffreytse/zsh-vi-mode#213]
Terminal program: iTerm2 3.4.19
 Operating system:  macOS 13.2
 ZSH framework: oh my zsh
 ZSH version: 5.8.1
 ZVM version: 0.8.5

## Basic examination

- [ x ] I have read through the [README](https://github.com/jeffreytse/zsh-vi-mode) page
- [ x ] I have the latest version of zsh-vi-mode
- [ x ] I have tested with another terminal program 

## Problem description

I recently replaced the computer and reinstalled some commonly used tools, but the Vi Mode Indicator does not display correctly after the zsh-vi-mode is installed, but each mode can be switched normally.
I'm using "spaceship" (oh-my-zsh theme), the same as my previous device, I've tried some other themes and they don't display the Vi Mode Indicator correctly.
I also tried the end point tool that comes with Mac OS and it worked the same way.

**Comment by jeffreytse** [comment]
Hi @Fengjing95 

Can you provide a screen record to show me the issue?

Thanks & Regards

**Comment by Fengjing95** [comment]
> 

The computer is at my home, I'm in the company now, and I'll send it out in 4 hours.
Thanks for your reply.

**Comment by Fengjing95** [comment]
> Hi @Fengjing95
> 
> Can you provide a screen record to show me the issue?
> 
> Thanks & Regards

I send a screen recording to your email.

**Comment by jeffreytse** [comment]
@Fengjing95 Are there any reproduction steps for me to dig this issue?

**Comment by Fengjing95** [comment]
@jeffreytse All my actions are as follows:
+ New computer activation
+ Install brew
+ Install oh-my-zsh
+ Install zsh-vi-mode with brew
+ Configure zshrc

Then I found this problem, and installed zsh-vi-mode (zsh-plugins, fig) in other ways, which still doesn't work.

**Comment by jeffreytse** [comment]
@Fengjing95 Could you please try the latest version, it means you clone from GitHub directly and source it.

**Comment by Fengjing95** [comment]
> @Fengjing95 Could you please try the latest version? It means you clone from GitHub repo directly and source it.

Install it in the form of a zsh plugin, is it the latest source code pulled?

**Comment by jeffreytse** [comment]
@Fengjing95 Yes, you can try to run `zvm_version` and see what's the version is.
